app/component/map/city-bike-popup.cjsx

```diff
@@ -0,0 +1,36 @@
+React                 = require 'react'
+Icon                  = require '../icon/icon.cjsx'
+StopCard              = require '../stop-cards/stop-card'
+MarkerPopupBottom     = require './marker-popup-bottom'
+NotImplementedLink    = require '../util/not-implemented-link'
+{FormattedMessage}    = require('react-intl')
+CityBikeContent     = require('../city-bike/city-bike-content')
+
+
+class CityBikePopup extends React.Component
+
+  addDesc: (station) =>
+    station.desc = station.name
+    station
+
+  render: ->
+
+    station = @addDesc @props.station
+
+    <div className="card">
+      <StopCard
```

Comment:
Is it really good idea to reuse stop card, or should we have a different card for bikeshare?